### PR TITLE
Feature/admin page

### PR DESCRIPTION
**Pull request by Larspolo**
## Description
Adds the possibility to disable the registration page and adds a privacy notice
image

### CODE REVIEW

File: src/django/VLE/views/instance.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import VLE.utils.responses as response
+from VLE.models import Instance
+from VLE.serializers import InstanceSerializer
+
+
+class InstanceView(viewsets.ViewSet):
+    """Instance view.
+
+    This class creates the following api paths:
+
```
Comment (Airgetfrog) [inline src/django/VLE/views/instance.py]: empty line

File: src/django/VLE/views/instance.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    """
+    def retrieve(self, request, pk=0):
+        """Get all instance details."""
+        try:
+            instance = Instance.objects.get(pk=1)
```
Comment (Airgetfrog) [inline src/django/VLE/views/instance.py]: are these supposed to be 1?

File: src/django/VLE/views/user.py [hunk 3]
```diff
@@ -103,6 +103,16 @@ def create(self, request):
         else:
             lti_id, user_image, is_teacher = None, None, False
 
+        if lti_id is None:
+            # Check is instance allows standable registration is user didnt register through some LTI instance
```
Comment (Airgetfrog) [inline src/django/VLE/views/user.py]: typos?

File: src/vue/src/views/Profile.vue [hunk 4]
```diff
@@ -3,7 +3,7 @@
         <bread-crumb>&nbsp;</bread-crumb>
         <b-card class="no-hover blue-border">
             <profile-data ref="profileData"/>
-            <notification-card/>
+            <!--<notification-card/>-->
```
Comment (Airgetfrog) [inline src/vue/src/views/Profile.vue]: ?

File: src/django/VLE/views/instance.py (hunk 2 above)
Comment (Larspolo) [reply src/django/VLE/views/instance.py]: yes, we only have 1 instance

File: src/vue/src/views/Profile.vue (hunk 4 above)
Comment (Larspolo) [reply src/vue/src/views/Profile.vue]: This is not working yet, so we dont want it in our deploy

**Pull request merged**